.stickler.yml

```diff
@@ -0,0 +1,4 @@
+linters:
+  rubocop:
+display_cop_names: true
+fixer: true
```

Comment:
Hi guys, this is a bit lazy not so? 😅 
While I totally understand you want this (I am also tempted), but it does not give you much room for feedback and learning. What you should have done is ignored certain files that are rails generated. That would have gotten rid of many linting errors and you could have fixed locally the linting errors that were reasonable to fix.

Your stickler file should look like following.

```yml
  
linters:
  rubocop:
    display_cop_names: true
files:
  ignore:
    - "bin/*"
    - "db/*"
    - "config/*"
    - "Guardfile"
    - "Rakefile"
```

How would you learn if your errors are being fixed for you automatically? Please remember to use this configuration next time.

Again, I totally understand your thinknig here, I'm just asking you to try it this way next time! 😄